packages/material-ui-lab/src/DesktopDateRangePicker/DesktopDateRangePicker.tsx

```diff
@@ -62,7 +62,7 @@ interface BaseDateRangePickerProps<TDate>
   value: RangeInput<TDate>;
 }
 
-const KeyboardDateInputComponent = DateRangePickerInput as React.FC<DateInputPropsLike>;
+const KeyboardDateInputComponent = (DateRangePickerInput as unknown) as React.FC<DateInputPropsLike>;
```

Comment:
What change made this fail?

Reply:
It is because `DateRangePickerInput` is not using `React.FC` anymore, so need to cast it to unknown first. I think this is fine since the previous implementation want to cast the type to `React.FC<DateInputPropsLike>` anyway.

![image](https://user-images.githubusercontent.com/18292247/118583082-a67fe180-b7be-11eb-9773-638449593683.png)